src/PeoplePage.1.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useState, useEffect } from 'react';
+import classNames from 'classnames';
+import { Person } from './types';
+import { PersonLink } from './PersonLink';
+import { Loader } from './components/Loader';
+import { getPeople } from './api';
+import { TableHeaders } from './types/TableHeaders';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const [errorMessage, setErrorMessage] = useState<string>('');
+  const [selectedPersonSlug, setSelectedPersonSlug] = useState<string>('');
+
+  const handleSelectPerson = (slug: string) => {
+    setSelectedPersonSlug(slug);
+    localStorage.setItem('selectedPerson', slug);
+  };
```

Comment:
Why do you need a local storage here?

Reply:
> Why do you need a local storage here?

I thought it was necessary for saving the link ,  after we reload the page.

Reply:
It is saved in url params, that is why we use it. Get rid in next iteration